build.gradle

```diff
@@ -50,3 +50,15 @@ compileTestJava {
 test {
     exclude 'de/schildbach/pte/live/**'
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = project.group
+            artifactId = archivesBaseName
+            version = project.version
+
+            from components.java
+        }
+    }
+}
```

Comment:
What is this good for? How is it related to the topic of this PR?

Reply:
This is needed for maven publish. Without this block Jitpack does not find the artifact.